scripts/install/install_macos.sh

```diff
@@ -507,8 +520,8 @@ install_package()
   info "Copying artifacts to install directory..."
   increase_indent
 
-  # This find command gets a list of all artifacts paths except config.yaml, logging.yaml, or opentelemetry-java-contrib-jmx-metrics.jar
-  FILES=$(cd "$TMP_DIR/artifacts"; find "." -type f -not \( -name config.yaml -or -name logging.yaml -or -name opentelemetry-java-contrib-jmx-metrics.jar \))
+  # This find command gets a list of all artifacts paths except logging.yaml, or opentelemetry-java-contrib-jmx-metrics.jar
+  FILES=$(cd "$TMP_DIR/artifacts"; find "." -type f -not \( -name -or -name logging.yaml -or -name opentelemetry-java-contrib-jmx-metrics.jar \))
```

Comment:
In think you could just remove the -name and -or here
```suggestion
  FILES=$(cd "$TMP_DIR/artifacts"; find "." -type f -not \( -name logging.yaml -or -name opentelemetry-java-contrib-jmx-metrics.jar \))
```

Reply:
🤦